routes.go

```diff
@@ -23,119 +23,132 @@ import (
 
 	"github.com/gorilla/mux"
 	"github.com/justinas/alice"
-	"github.com/prometheus/client_golang/prometheus"
 	"github.com/xmidt-org/argus/store"
-	"github.com/xmidt-org/themis/xhealth"
-	"github.com/xmidt-org/themis/xhttp/xhttpserver"
-	"github.com/xmidt-org/themis/xmetrics"
-	"github.com/xmidt-org/themis/xmetrics/xmetricshttp"
+	"github.com/xmidt-org/arrange"
+	"github.com/xmidt-org/arrange/arrangehttp"
+	"github.com/xmidt-org/candlelight"
+	"github.com/xmidt-org/httpaux"
+	"github.com/xmidt-org/touchstone/touchhttp"
+	"go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux"
 	"go.uber.org/fx"
 )
 
-type ServerChainIn struct {
+type PrimaryRouterIn struct {
 	fx.In
+	Router    *mux.Router `name:"server_primary"`
+	APIBase   string      `name:"api_base"`
+	AuthChain alice.Chain `name:"auth_chain" optional:"true"`
+	// Tracing will be used to set up tracing instrumentation code.
+	Tracing  candlelight.Tracing `optional:"true"`
```

Comment:
the way we use it here, tracing will always be provided so we can drop the optional tag

Reply:
sounds good - will make the change, but I expect this to move in the near future with this issue: https://github.com/xmidt-org/argus/issues/184